src/serve_rendered.js

```diff
@@ -1414,82 +1465,142 @@ export const serve_rendered = {
 
     const queue = [];
     for (const name of Object.keys(styleJSON.sources)) {
+      let source_type;
       let source = styleJSON.sources[name];
       const url = source.url;
 
-      if (url && url.lastIndexOf('mbtiles:', 0) === 0) {
+      if (
+        url &&
+        (url.lastIndexOf('pmtiles:', 0) === 0 ||
+          url.lastIndexOf('mbtiles:', 0) === 0)
+      ) {
         // found mbtiles source, replace with info from local file
         delete source.url;
 
-        let mbtilesFile = url.substring('mbtiles://'.length);
+        let inputFile = url.replace('pmtiles://', '').replace('mbtiles://', '');
         const fromData =
-          mbtilesFile[0] === '{' && mbtilesFile[mbtilesFile.length - 1] === '}';
+          inputFile[0] === '{' && inputFile[inputFile.length - 1] === '}';
```

Comment:
and here you can do `inputFile.startsWith('{') && inputFile.endsWith('}')`

Reply:
I updated this